contracts/minter/src/contract_tests.rs

```diff
@@ -82,17 +79,15 @@ fn setup_whitelist_contract(
             "whitelist",
             None,
         )
-        .unwrap();
-
-    Ok(whitelist_addr)
+        .unwrap()
 }
 
 // Upload contract code and instantiate sale contract
 fn setup_minter_contract(
     router: &mut StargazeApp,
     creator: &Addr,
     num_tokens: u64,
-) -> Result<(Addr, ConfigResponse), ContractError> {
+) -> (Addr, ConfigResponse) {
```

Comment:
Is the result wrapper removed because it's not needed in testing?

Reply:
Two reasons:
1. There's no errors thrown by this function, so the function signature doesn't need an error.
2. In tests you want to catch errors because they have nowhere to bubble up to.